tests/functional/disaster-recovery/regional-dr/test_acm_kubevirt_dr_integration.py

```diff
@@ -0,0 +1,331 @@
+import logging
+
+from time import sleep
+
+from ocs_ci.deployment.cnv import CNVInstaller
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import skipif_ocs_version
+from ocs_ci.framework.pytest_customization.marks import (
+    rdr,
+    turquoise_squad,
+    tier2,
+)
+from ocs_ci.helpers import dr_helpers
+from ocs_ci.helpers.cnv_helpers import run_dd_io
+from ocs_ci.helpers.dr_helpers import wait_for_all_resources_deletion
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.acm.acm import AcmAddClusters
+from ocs_ci.helpers.dr_helpers_ui import (
+    assign_drpolicy_for_discovered_vms_via_ui,
+    verify_drpolicy_ui,
+)
+from ocs_ci.ocs.dr.dr_workload import validate_data_integrity_vm
+from ocs_ci.ocs.node import get_node_objs, wait_for_nodes_status
+from ocs_ci.ocs.resources.drpc import DRPC
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.utility.utils import ceph_health_check
+
+logger = logging.getLogger(__name__)
+
+
+@rdr
+@tier2
+@turquoise_squad
+@skipif_ocs_version("<4.19")
+class TestACMKubevirtDRIntergration:
+    """
+    Test ACM Kubevirt DR Integration by DR Protecting Discovered VMs via VMs page of ACM UI as Standalone
+    and Shared Protection type and perform DR operation on them- RHSTOR-6413
+
+    """
+
+    # @pytest.mark.polarion_id("OCS-xxxx")
+    # TODO: Add Polarion ID when available
+    def test_acm_kubevirt_using_shared_protection(
+        self,
+        setup_acm_ui,
+        discovered_apps_dr_workload_cnv,
+        nodes_multicluster,
+        node_restart_teardown,
+    ):
+        """
+        DR operation on discovered VMs using Shared Protection type, both VMs are tied to a single DRPC
```

Comment:
It would be nice to see test steps updated here as part of documentation for better readability.

Reply:
Updated!